core/src/main/java/org/bitcoinj/core/BlockChain.java

```diff
@@ -51,42 +51,27 @@ public class BlockChain extends AbstractBlockChain {
      * {@link MemoryBlockStore} if you want to hold all headers in RAM and don't care about
      * disk serialization (this is rare).</p>
      */
-    public BlockChain(Context context, Wallet wallet, BlockStore blockStore) throws BlockStoreException {
```

Comment:
We might want to consider deprecating these constructors, but personally I'm fine with just removing them. Good riddance!